timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -680,7 +680,7 @@ private static LintFix quickFixIssueTagLength(UExpression argument, String tag)
         .build();
   }
 
-  private static LintFix quickFixIssueExceptionLogging(UExpression arg2) {
+  private LintFix quickFixIssueExceptionLogging(UExpression arg2) {
```

Comment:
Why?

Reply:
`Detector.fix()` changed to non-static, requiring changes up the local call stack.